static/scss/search-page.scss

```diff
@@ -56,6 +56,11 @@
       padding: 0 10px 0 23px;
     }
 
+    .sk-hierarchical-menu-list {
+      max-height: 150px;
+      overflow-y: scroll;
```

Comment:
Yeah, this just sets how an overflow should be handled. Scrollbar display is left up to the browser. most these days do not display a scrollbar all the time, and I think all browsers on macOS will use a system scrollbar, which by default is not persistant. getting a persistant, cross-platform scrollbar unfortunately requires using a JS reimplementation of the scrolling already implemented by the browser, which is both yucky and unfortunate.